README.md

```diff
@@ -1,13 +1,17 @@
 # Nautobot
 
-![Nautobot](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/nautobot_logo.svg "Nautobot logo")
+![Nautobot](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/nautobot_logo.svg "Nautobot logo"). Nautobot is a Network Source of Truth and Network Automation Platform. Nautobot was initially developed as a fork of NetBox (v2.10.4).  NetBox was originally developed by Jeremy Stretch at DigitalOcean and the NetBox Community. Nautobot runs as a web application atop the [Django](https://www.djangoproject.com/) Python framework with a
+[PostgreSQL](https://www.postgresql.org/) or [MySQL](https://www.mysql.com) database.
 
-Nautobot is a Network Source of Truth and Network Automation Platform.
```

Comment:
I thinks the reformatting here isn't entirely helpful.

But @nautobot/core could we consider moving the fork portion of this down to the footer, something like:

```md
<!--(the rest of the README.md file)-->

---
## Notices

> Nautobot was initially developed as a fork of NetBox (v2.10.4).  NetBox was originally developed by Jeremy Stretch at DigitalOcean and the NetBox Community. 

```